### DISCUSSION TITLE
NVIDIA Kiosk 點餐機

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#151] [General]
NVIDIA開發最新Service Kiosk，只要對著Kiosk點餐機說出你要的餐點，不用輸入就能自動完成點餐到結帳。

Kiosk上有個AI虛擬助理，不只聽得懂你要點的起司漢堡是什麼口味，還會根據不同客人來推薦套餐或適合加哪些配料，甚至顧客要查詢食物的蛋白質含量，或想要素食漢堡也難不倒它。

Customer Service Kiosk :
https://www.youtube.com/watch?v=ox2Cc88I-Os

**Comment by enoch-lightning** [comment]
以後可能都沒有負責點餐的店員，AI正在快速改變我們的生活方式。

**Reply by enoch-lightning** [reply]
🌭 🍔 🍟 🍕 🥪 🥙 🧆 🌮 🌯 🍿 🍤

**Comment by chia-kai-chang** [comment]
好生動的點餐機器人阿~

最近Google在推廣Google Actions:
1. https://developers.google.com/assistant/console
2. https://www.youtube.com/watch?v=g410vzUXiyQ&ab_channel=Google

是由Google Cloud去認知使用者的意圖。

如果把這些服務堆疊起來就可以做一個很棒的服務機器人了。

**Comment by xCk27x** [comment]
[麥當勞聯手 IBM 欲打造全 AI 得來速點餐系統](https://www.inside.com.tw/article/25379-mcdonalds-ibm-drive-thru)
麥當勞也打算要推出AI得來速，好像從2019初就開始計畫了

**Comment by chupochun** [comment]
以AI店員為餐廳的主題或許也是一種不錯的行銷手法

**Comment by bloodtrail0511** [comment]
科技真的在快速改變我們的生活，兩三年前電子菜單還是個難得一見的新潮東西呢

**Comment by yucheng1028** [comment]
這樣機器人是不是就會紀錄我這周吃了多少垃圾食物了....
